db/schema.rb

```diff
@@ -358,6 +358,16 @@
     t.index ["reset_password_token"], name: "index_users_on_reset_password_token", unique: true, using: :btree
   end
 
+  create_table "web_push_subscriptions", force: :cascade do |t|
+    t.integer  "account_id"
```

Comment:
This was generated by the following command, so I cannot answer that 😄  How should it look like?
```
rails generate model WebPushSubscription account_id:integer endpoint:string:unique key_p256dh:string key_auth:string
```